doc/2/guides/introduction/1-what-is-kuzzle/index.md

```diff
@@ -0,0 +1,98 @@
+---
+code: false
+type: page
+title: What is Kuzzle
+description: Why are we spending so much time developing this backend?
+order: 100
+---
+
+# What is Kuzzle
+
+If you're on this page it's probably because you need a backend for your mobile, web or IoT application.
+
+Once again you had been preparing to develop a backend from scratch... Well, maybe not entirely from scratch, because you probably planned to use some kind of framework (and there are a lot of them!) to make it easier for you.  
+Those frameworks allow you to develop faster by providing a predefined structure, classes and configurations.
+
+However, you will still have to develop the majority of the basic features:
+ - storing and searching data
+ - permission management
+ - user authentication
+ - access to data and functionalities through an API
+
+Each of these features will take time. Time to develop but also time to:
+ - debug
+ - test 
+ - maintain
+ - secure
+
+In short, once again you were going to spend a lot of time on code that doesn't bring any value to your users but is nevertheless essential.
+
+This time could have been used for many other things:
+ - development of business functionalities
+ - UI / UX of frontend applications
+ - 100% coverage by automated tests
+ - implementation of devops best practices
+ - marketing of your product
+ - ...
+
+It is on the basis of this failure to optimize development time that we decided to start developing Kuzzle 5 years ago and that we have been devoting all our efforts to it ever since.
+
+## How it works
+
+Kuzzle is a backend with ready-to-use features that can be extended in the same way as any framework.
+
+When you start Kuzzle, you automatically have access to an API exposing the following features:
+ - Data storage and access
+ - Advanced permission system
+ - Multi authentication
+ - Multi protocol API (Http, WebSocket, MQTT)
+ - Realtime engine
+ - Integrated cluster mode
+ - and more!
+
+
+## Complete ecosystem
+
+In addition to Kuzzle, we are developing many other projects to facilitate the use of our backend.   
+
+All these projects are also available under the Apache-2 license on [Github](https://github.com/kuzzleio).
+
+### Admin Console
+
+The [Admin Console](/core/2/some-link) is a Single Page Application (SPA) written in Vue.js.  
+
+It is used to manage its data and the user permissions system.
+
+As it is a SPA, no data related to your Kuzzle application will pass through our servers, so you can use the online version available at [http://console.kuzzle.io](http://console.kuzzle.io).
```

Comment:
```suggestion
As it is a single-page application (SPA), no data related to your Kuzzle instances will pass through our servers, so you can use the online version available at [http://console.kuzzle.io](http://console.kuzzle.io).
```

Reply:
Feel free to use either **instances** or **instance** on this one. (or is **application** a new choice of kuzzle definition)

Reply:
Application is better :+1: